recipe/bld-arrow.bat

```diff
@@ -46,8 +46,18 @@ cmake -G "Ninja" ^
       -DCMAKE_UNITY_BUILD=ON ^
       %EXTRA_CMAKE_ARGS% ^
       ..
-if errorlevel 1 exit 1
+if errorlevel 1 (
+	goto :build
+) else (
+	goto :cfgfail
+)
+
+:cfgfail
+type CMakeFiles\CMakeOutput.log
+type CMakeFiles\CMakeError.log
+exit 1
 
+:build
```

Comment:
There's probably a better way to accomplish this, but I lack a Windows machine to test this on. Would welcome input from others :)